scripts/typecheck_tests.py

```diff
@@ -70,7 +70,6 @@
         'List item 0 has incompatible type "Type[',
         'error: Module has no attribute "coreapi"',
         'Value of type "Optional[str]" is not indexable',
-        "Module 'rest_framework.compat' has no attribute 'yaml'",
```

Comment:
So this did cause errors for `rest_framework.compat.yaml`.

Why didn't it cause errors for `rest_framework.compat.coreapi` etc?  🤔